Feat/app/UI

ljhyeon opened the pull request:
웹 버전 UI 구현했습니다.
- supply(main)에서 dialog는 값을 입력해야하는지, 정보만 표출하는 건지 확실치 않아 아직 구현하지 않았습니다.
- setting에서 내 정보 란에 '주소' 정보가 뜨는데, 해당 정보를 수집하지 않아서 이에 대한 개선방안이 필요합니다.
- 여전히 로그아웃을 할 수가 없습니다...
--
25.07.05
- dialog 관련 내용 확인했고, 정보 표출용으로 구현했습니다. (접수 가능)
- api 연결 보다 직접 입력할 수 있는 dialog로 대체했습니다.
- 로그아웃은 로고 누르면 할 수 있게 했고, home에 우측 상단 버튼을 추가했습니다.
- 하단바 라우팅 문제가 발생해 store 추가해서 해결했습니다.
  - store에 사용자 기부 품목까지 함께 저장해 표출할 수 있게 했습니다.

ljhyeon committed:
feat: 로그인 UI 구성

ljhyeon committed:
feat: 회원등록 UI 구성

ljhyeon committed:
feat: 홈 UI 구성

ljhyeon committed:
feat: supply ui 구성

- dialog는 기획 구체화되면 추가 개발 예정

ljhyeon committed:
feat: status ui 구성

ljhyeon committed:
feat: setting ui 구성

hyeok8055 commented:
1. 주소입력 API 쓰는건 어떨까요 예전에 써봤을때에는 크게 어렵진 않았던것같은데
2. 로그아웃은.... 이번엔 뭘 눌러야 로그아웃 시키면 되려나

ljhyeon committed:
feat: supply 모달 구현

ljhyeon committed:
feat: 로그아웃 추가

ljhyeon committed:
feat: 주소 입력 기능 추가

ljhyeon committed:
fix: store 제작 및 라우팅 오류 수정

- shelterStore를 통해 하단바로 인한 라우팅 오류 해결
- shetlerStore에 기부 품목까지 함께 저장해 main(supply)에서 기부하기로 결정하면 기부 배송(status)에 반영됨
  - 현 상황은 중복 등록 가능
